packager/media/file/udp_file_posix.cc

```diff
@@ -188,8 +188,12 @@ bool UdpFile::Open() {
   local_sock_addr.sin_port = htons(dest_port);
   local_sock_addr.sin_addr.s_addr  = htonl(dest_addr);
   // We do not need to require exclusive bind of udp sockets
-  int optval = 1;
-  setsockopt(new_socket.get(), SOL_SOCKET, SO_REUSEADDR, &optval, sizeof(optval));
+  const int optval = 1;
+  if (setsockopt(new_socket.get(), SOL_SOCKET, SO_REUSEADDR, &optval, sizeof(optval)) < 0)
```

Comment:
We have a 80-characters limit per line; also, "{" should not be in its own line. Can you change it to:

```
  if (setsockopt(new_socket.get(), SOL_SOCKET, SO_REUSEADDR, &optval,
                 sizeof(optval)) < 0) {
    LOG(ERROR)
        << "Could not apply the SO_REUSEADDR property to the UDP socket.";
    return false;
  }
```